cylc/flow/cfgspec/globalcfg.py

```diff
@@ -785,8 +785,12 @@
                       be appropriate if following the pattern
                       ``hosts = main, backup, failsafe``.
                 ''')
-        with Conf('localhost', meta=Platform):
+        with Conf('localhost', meta=Platform, desc='''
+            A default platform defining settings for the local computer.
```

Comment:
Purely a suggestion and not a blocker to merge (equally happy for you to ignore or correct me!)...
Wondering if this description could be an opportunity to explain to users that, if they use cylc to submit to a cylc server, the server is the localhost, since certainly here at the MO, it is potentially a bit confusing grasping the idea of the server being the localhost, rather than the machine they are on.
Perhaps an extra sentence after this one to explain?